src/langcheck/metrics/en/__init__.py

```diff
@@ -6,6 +6,9 @@
     harmful_activity,
     hate_speech,
     personal_data_leakage,
+    summarization_quality,
```

Comment:
Now `langcheck.metrics.*` is the alias to `langcheck.metrics.en.*`. Could you also add the metric names to `src/langcheck/metrics/__init__.py`?

Reply:
Fixed! [3fd6b5f](https://github.com/citadel-ai/langcheck/pull/197/commits/3fd6b5f599c471b80a29bdf9ab2228cc91575282)